### ISSUE TITLE
Table theming issues

### ISSUE THREAD

**Issue by skwerlman** [StylishThemes/Wikipedia-Dark#28]
Bright greys:
![image](https://user-images.githubusercontent.com/6954107/30850436-87eef1ac-a273-11e7-9b08-10d4741073a1.png)
A weird grey bar:
![image](https://user-images.githubusercontent.com/6954107/30850470-a27c96e6-a273-11e7-9d33-f4733531ec85.png)

https://en.wikipedia.org/wiki/Myst_(series)

**Comment by Mottie** [comment]
Hi @skwerlman!

If you look at the [limitations](https://github.com/StylishThemes/Wikipedia-Dark#limitations) section of the readme, you'll see a note about how Wikipedia uses inline styles to add background colors to table cells. There are only two solutions that I can think of:

1. Ignore all custom table cell colors and override them all.
2. Add every single color combination, full and shorthand hex colors, with and without spaces, and include `th` and `td` cells.

The first would be easy, but not pretty. The second... well, I don't have the time to do something like that. My OCD would be relieved, but my wife would hate me.

The "weird grey bar" is because that's how the creator of the page added padding:

```html
<div style="font-size:90%;float:right;border-left:1em solid white;">...</div>
```

This style is targeting the "white" border... I guess I could force the width to be 1px, but then I would have to add a margin. And I don't even want to think about how that would effect other pages. There are no classes or easy methods to fix this problem. Maybe someone needs to edit the wiki page and fix it?

TLDR; There's not much I can do with this userstyle.

**Comment by Mottie** [comment]
Hmm, ok maybe I can fix this one... it's the `tr` with the style, not the `td`... I guess I should have looked at the source before replying.

And I edited the wiki page... YAY for padding.

**Comment by hartman** [comment]
I've made some [further changes](https://en.wikipedia.org/w/index.php?title=Myst_(series)&diff=802492402&oldid=802483246) to update the table to be better inline with our current standards. That thing was old....

**Comment by skwerlman** [comment]
Thanks for getting this figured out and fixed :heart:

**Comment by Mottie** [comment]
I just fixed the diff page styling as well 😸.
